Library/Formula/jasper.rb

```diff
@@ -23,11 +24,56 @@ class Jasper < Formula
     cause "Undefined symbols when linking"
   end
 
+  # CVE-2011-4516: heap-based buffer overflow
+  # CVE-2011-4517: heap-based buffer overflow
+  # CVE-2014-8137: double-free
+  # CVE-2014-8138: heap-based buffer overflow
+  # CVE-2014-8157: off-by-one
+  # CVE-2014-8158: multiple stack-based buffer overflows
+  # CVE-2014-9029: multiple off-by-one
```

Comment:
Would be good to submit to https://github.com/homebrew/patches instead of Gists too.